apps/fz_http/lib/fz_http/users/user.ex

```diff
@@ -37,8 +37,6 @@ defmodule FzHttp.Users.User do
     user
     |> cast(attrs, [
       :role,
-      :sign_in_token,
-      :sign_in_token_created_at,
```

Comment:
system generated properties should be avoided from `cast`

Reply:
Awesome, I opened firezone/backlog#358 to get all these changed.